README.md

```diff
@@ -100,7 +110,8 @@ SQL database. If you need an example, you can use: people and wishes
 
 Your answer:
 ```
-Replace this with your answer
+Schema is a diagram where explains relationship between data entities.
+Each people can have many wishes.
```

Comment:
Schema can provide information about relationships but are they limited to that? Do we have a schema in a database with only one table?